Capfile

```diff
@@ -35,6 +35,7 @@ require "capistrano/rails/migrations"
 # require "capistrano/passenger"
 
 require "capistrano/yarn"
+require 'capistrano/sidekiq'
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->